planning/route_handler/src/route_handler.cpp

```diff
@@ -911,6 +911,73 @@ lanelet::Lanelets RouteHandler::getRightOppositeLanelets(
   return lanelet_map_ptr_->laneletLayer.findUsages(lanelet.rightBound().invert());
 }
 
+lanelet::ConstLanelets RouteHandler::getAllLeftSharedLinestringLanelets(
+  const lanelet::ConstLanelet & lane, const bool & include_opposite) const noexcept
+{
+  lanelet::ConstLanelets shared;
```

Comment:
@zulfaqar-azmi-t4 
Can you change "shared" to "shared_line_string" or other naming which is not reserved word?

Reply:
changing it to `linestring_shared`. Will be available in the next commit.